src/sentry/api/bases/group.py

```diff
@@ -70,7 +69,10 @@ def convert_args(self, request, issue_id, organization_slug=None, *args, **kwarg
 
         with configure_scope() as scope:
             scope.set_tag("project", group.project_id)
-            scope.set_tag("organization", group.project.organization_id)
+
+        # we didnt bind context above, so do it now
```

Comment:
Nit: We could just make this call in one place
```
organization = organization if organization is not None else group.project.organization
bind....
```

Reply:
only reason I did this was then if get_group_with_direct/permissions errors out, it already has the organization context available (given we dont yet have a goup)